socialwarehouse/agents/management/commands/load_fec_attestations.py

```
@@ -0,0 +1,101 @@
+                           {attested_values}, md5(({attested_values})::text),
+                           (g._synced_at at time zone 'UTC'), NULL,
+                           'unknown', 0, true, now(), now()
+                    FROM gold.{table} g
+                    WHERE {where}
+                    ON CONFLICT (entity_uuid) DO UPDATE SET
+                        attested_values = EXCLUDED.attested_values,
+                        attested_values_hash = EXCLUDED.attested_values_hash,
+                        attested_at = EXCLUDED.attested_at,
+                        attestation_source_tier = EXCLUDED.attestation_source_tier,
+                        updated_at = now()
+                """)
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)